cmake/Macros.cmake

```diff
@@ -247,12 +272,13 @@ macro(sfml_add_example target)
 endmacro()
 
 
-# Find the requested package and make an INTERFACE library from it
-# Usage: sfml_find_package(wanted_target_name
-#                          [INCLUDE "OPENGL_INCLUDE_DIR"]
-#                          [LINK "OPENGL_gl_LIBRARY"])
-function(sfml_find_package)
-    set(CMAKE_MODULE_PATH "${PROJECT_SOURCE_DIR}/cmake/Modules/")
+# Create an interface library for an external dependency. This virtual target can provide
+# link specifications and include directories to be used by dependees.
+# The created INTERFACE library is tagged for export to be part of the generated SFMLConfig
+# Usage: sfml_add_external(target_name
+#                          [INCLUDE "extlibs/include"]
+#                          [LINK "extlibs/libfoo/libfoo.a"])
+function(sfml_add_external)
```

Comment:
Is this required because the findGLES cmake module doesn't work, or some other reason?

Reply:
Originally there was no use of the FindOpenGLES module, I don’t even know if such module exists. But the point is rather that there is nothing to find. For example on iOS when linking against OGLES you just use "-framework OpenGLES", because it’s provided by the SDK so you are sure it will always be found and you’ll never need to search in other locations. So instead of searching for it then creating an interface library with the found information, we directly create that interface library with the flags we already know.

Reply:
This is the same as the OpenAL framework, which is included in the SDK, but we find using the cmake module. I'm not super fussed about it to be honest, just looking for the sake of simplicity/cleanliness